packages/server/lib/modes/run.js

```diff
@@ -1650,6 +1650,7 @@ module.exports = {
   async run (options, loading = Promise.resolve()) {
     if (require('../util/electron-app').isRunningAsElectronProcess({ debug })) {
       const app = require('electron').app
+      const { clearCtx } = require('@packages/data-context')
```

Comment:
Was there a reason to inline this require instead of putting it at the top?

Will make an eventual ESM convert easier.

Reply:
The existing data-context require used to be at root but was changed with https://github.com/cypress-io/cypress/pull/20932/files#diff-58fd4a8e28c13b52b19c42346eade66ed1de5a8b139541dff4e94fef42c3c701R666-R667. I assume due to how the run script is getting loaded?

Regardless, I am going to revert the changes to this file for run mode. I don't think the process.exit handler is deterministic here, just like it wasn't before in the ProjectLifecycleManager. And because we don't initialize watchers in run mode, the teardown isn't necessary prior to exit.